db/migrate/20190813142450_change_tribunal_org_type.rb

```diff
@@ -1,7 +0,0 @@
-class ChangeTribunalOrgType < ActiveRecord::Migration[5.1]
```

Comment:
This has already run on integration and staging.  It might be OK as long as the data sync from production runs and overwrites those databases, though it might be better to write another migration that reverses this one. That would ensure that the data is in sync with the reverted code.

Reply:
Makes sense, I'll do that

Reply:
@sihugh does it look better now?

Reply:
Thanks, that's great 👍